### ISSUE TITLE
argh, a bug in caching of some sort

### ISSUE THREAD

**Issue by SunnyMarkLiu** [JohnLangford/vowpal_wabbit#1247]
I was use vw command line to train a simple model, but I got this error:
`vw example #575348(parser.cc:272): argh, a bug in caching of some sort!`
How can I solve this?
Thanks.
SunnyMarkLiu

**Comment by JohnLangford** [comment]
Can you create a simple reproducible test case?

-Johm

**Comment by MLWave** [comment]
Try adding `-k` parameter during training. It kills any existing cache that may be incompatible with your new parameters.

**Comment by SunnyMarkLiu** [comment]
Thanks

**Comment by ishanshrivastava2011** [comment]
Adding -k did not work for me. Is there anything else I can try.
This is the code I ran.
vw house_dataset -l 10 -c --passes 25 --holdout_off -k
 
This is how the douse_dataset file looks:
0 |u price:.23 sqft:.25 |v age:.05 2006
1 2 'second_house |u  price:.18 sqft:.15 |v age:.35 1976
0 1 0.5 'third_house |u  price:.53 sqft:.32 |v age:.87 1924

**Comment by MLWave** [comment]
Can not replicate. Can you run the below (including the version command) and post the output here?

house_dataset.vw:
`0 |u price:.23 sqft:.25 |v age:.05 2006
1 2 'second_house |u price:.18 sqft:.15 |v age:.35 1976
0 1 0.5 'third_house |u price:.53 sqft:.32 |v age:.87 1924`

run:
`vw house_dataset.vw -l 10 -c --passes 25 --holdout_off -k`

output:
`[...] average loss = 0.057188 [...]`

version:
`vw --version`

output:
`8.1.1`
